django/core/management/commands/runserver.py

```diff
@@ -108,22 +108,9 @@ def run(self, **options):
         else:
             self.inner_run(None, **options)
 
-    def inner_run(self, *args, **options):
-        # If an exception was silenced in ManagementUtility.execute in order
-        # to be raised in the child process, raise it now.
-        autoreload.raise_last_exception()
-
-        threading = options['use_threading']
-        # 'shutdown_message' is a stealth option.
-        shutdown_message = options.get('shutdown_message', '')
+    def on_bind(self, server_name, server_port):
```

Comment:
Hello,
Isn't the (*args, **options) format more consistent with current Django codes?

Reply:
I don't think so, but it should be better to be a private method, because nobody else should call it

Reply:
Aha,
yes, I agree with you on the private method solution.

Reply:
I have fixed that using rebase (on a newer commit) to edit my commit and then push my changes by force; I hope that's ok

Reply:
> I hope that's ok

No problem.

@fmwviormv Could I ask you to fill out [a ticket](http://code.djangoproject.com/newticket)? It's quick--we use it for task tracking and to verify that the proposed changes sound good.

Reply:
I've created a ticket: https://code.djangoproject.com/ticket/32813